up-and-running/platforms/other.md

```diff
@@ -1,3 +1,68 @@
-# Using Other Platforms
+# Other Linux Platforms
 
-## Summary
+npm Enterprise works on most modern versions of Linux:
+
+- Ubuntu 14.04 / 15.10
+- CentOS 7.x
+- Red Hat Enterprise Linux (RHEL) 7.x
+- Debian 7.7
+
+Simply follow the steps listed below:
+
+## Step 1: Install npm and Node.js
+
+Now that you have a server provisioned, let's install npm and Node.js
+
+_On Ubuntu:_
+
+```bash
+curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
```

Comment:
I think I'll stick with Node 4, which is the LTS and I've been testing with in a few places; I'm totally cool with moving over to 6.x though in future documentation.